README.md

```diff
@@ -204,6 +205,13 @@ and so locking affects them as well.
 
 To change a partially locked controller, set the `fullText` property.
 
+<details>
+  <summary>Epilepsy warning! Example gif:</summary>
+
+
+![cleaner-readonly](https://user-images.githubusercontent.com/79095619/213762470-683e6304-681f-4292-80a2-67d8ec88cde6.gif)
+  
+</details>
```

Comment:
1. This expander is GitHub specific, it may not work on all readers, not sure about pub.dev.
2. We do not want to identify with unhealthy conditions, so any mentions should be removed.

So let's always show a gif that is slow enough to not distract. May hit fewer keys, may show each key for longer time. This can be achieved by editing the gif if key caption TTL cannot be changed at the recording time.